cdap-explore/src/main/java/co/cask/cdap/explore/guice/ExploreRuntimeModule.java

```diff
@@ -227,11 +233,15 @@ protected void configure() {
         LOG.info("Setting {} to {}", HiveConf.ConfVars.LOCALSCRATCHDIR.toString(),
                  System.getProperty(HiveConf.ConfVars.LOCALSCRATCHDIR.toString()));
 
-
-        // We don't support security in Hive Server.
-        System.setProperty("hive.server2.authentication", "NONE");
-        System.setProperty("hive.server2.enable.doAs", "false");
-        System.setProperty("hive.server2.enable.impersonation", "false");
+        if (UserGroupInformation.isSecurityEnabled()) {
+          System.setProperty("hive.server2.authentication", "NOSASL");
```

Comment:
None of the three properties here might be required. Can you verify? I think the three properties in `else` block might also not be required.